test/contrib/s3_test.py

```diff
@@ -454,15 +453,17 @@ def test_copy_dir(self):
             self.assertTrue(s3_client.exists(file_path))
 
         s3_dest = 's3://mybucket/copydir_new/'
-        s3_client.copy(s3_dir, s3_dest, threads=10, part_size=copy_part_size)
+        response = s3_client.copy(s3_dir, s3_dest, threads=10, part_size=copy_part_size)
```

Comment:
I'd have a test for empty directory and empty file cases.

Reply:
There already exists a `test_copy_empty_file`, but I'll add a test for empty directory